src/components/views/rooms/EventTile.tsx

```diff
@@ -470,7 +470,11 @@ export class UnwrappedEventTile extends React.Component<EventTileProps, IState>
         this.threadState?.off(NotificationStateEvents.Update, this.onThreadStateUpdate);
     }
 
-    public componentDidUpdate(prevProps: Readonly<EventTileProps>) {
+    public componentDidUpdate(prevProps: Readonly<EventTileProps>, prevState: Readonly<IState>) {
+        // If the verification state changed, the height might have changed
+        if (prevState.verified !== this.state.verified) {
+            this.props.onHeightChanged();
+        }
```

Comment:
This is (imho) a simplification (and in line with react's recommendations), but also turns out to be needed to keep some other tests passing, to ensure we don't call onHeightChanged if the verified state stays at `null`.